manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -35,3 +35,16 @@ rules:
   - console-public
   verbs:
   - get
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
+  namespace: openshift-config
```

Comment:
This should be a different file, `ns-openshift-config.yaml`, since its a different namespace than `ns-openshift-config-managed.yaml`.  The `-ns-` indicates the namespace.

Reply:
I don't know if this is exactly what you will need with the `syncer`.  I imagine you might need `list`, `watch` as well.